presto-main/src/main/java/com/facebook/presto/testing/InterfaceTestUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.testing;
+
+import java.lang.reflect.Method;
+
+import static java.lang.String.format;
+
+public class InterfaceTestUtils
```

Comment:
`final` (for whatever reason we add this in utility classes even though the private ctor does the trick)